api/dispatcher.proto

```diff
@@ -48,6 +49,24 @@ service Dispatcher { // maybe dispatch, al likes this
 	rpc Tasks(TasksRequest) returns (stream TasksMessage) {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-worker" roles: "swarm-manager" };
 	};
+
+	// CreateContainerAttachment allows the node to request the resources
+	// allocation needed for a container attachment on the specific node.
+	rpc CreateContainerAttachment(CreateContainerAttachmentRequest) returns (CreateContainerAttachmentResponse) {
+		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-worker" roles: "swarm-manager" };
+	};
+
+	// RemoveContainerAttachment allows the node to request the release of
+	// the resources associated to the container attachment.
+	rpc RemoveContainerAttachment(RemoveContainerAttachmentRequest) returns (RemoveContainerAttachmentResponse) {
+		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-worker" roles: "swarm-manager" };
+	};
+
+	// ContainerAttachments is a stream of container attachment resources
+	// allocated for the node.
+	rpc ContainerAttachments(ContainerAttachmentsRequest) returns (stream ContainerAttachmentMessage) {
```

Comment:
As soon as we finalize on the renaming for `ContainerAttachment`, will check this again.